ManageWikiSettings.php

```diff
@@ -2617,6 +2617,15 @@
 		'help' => 'When enabled, this shows Popups by default.',
 		'requires' => [],
 	],
+	'wgScratchBlocks4BlockVersion' => [
+		'name' => 'Default Scratch Blocks version',
+		'from' => 'scratchblocks',
+		'type' => 'integer',
```

Comment:
This should be float if it's a decimal I think

Reply:
Both are supported by the extension.

Reply:
Then it should be set to just 3 at default.

Reply:
I set this to 3 at first as that is what anybody familiar with the extension should expect to see but this is minor so done.